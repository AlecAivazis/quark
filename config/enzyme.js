// external imports
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import browserEnv from 'browser-env'

// add a request animation frame polyfill
import raf from 'raf'
raf.polyfill()

// enable the react 16 adapter for enzyme
Enzyme.configure({ adapter: new Adapter() })

// add various browser mocks
browserEnv()
