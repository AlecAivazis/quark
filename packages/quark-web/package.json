{
    "name": "quark-web",
    "version": "1.0.9-0",
    "module": "index.js",
    "main": "index.js",
    "dependencies": {
        "quark-core": "^1.0.8",
        "react-markdown": "^3.3.0"
    }
}
