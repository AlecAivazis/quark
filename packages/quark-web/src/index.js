// local exports
export * from 'quark-core'
