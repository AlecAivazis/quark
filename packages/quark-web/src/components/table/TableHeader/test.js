// external imports
import React from 'react'
import { Text } from 'react-native'
import { mount } from 'enzyme'
// local imports
import { TableHeader } from 'quark-web'

it('renders', () =>
    expect(
        mount(
            <TableHeader>
                <Text>hello</Text>
            </TableHeader>
        )
    ).toMatchSnapshot())
