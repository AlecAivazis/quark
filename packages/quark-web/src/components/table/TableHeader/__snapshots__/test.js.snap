// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<TableHeader>
  <GetTheme>
    <thead
      style={
        Object {
          "borderBottom": "3px solid",
          "borderColor": "#48484D",
        }
      }
    >
      <tr
        style={Object {}}
      >
        <Text>
          <div
            className="rn-borderTopWidth-13yce4e rn-borderRightWidth-fnigne rn-borderBottomWidth-ndvcnb rn-borderLeftWidth-gxnn5r rn-boxSizing-deolkf rn-color-homxoj rn-display-1471scf rn-font-1lw9tu2 rn-fontFamily-14xgk7a rn-fontSize-1b43r93 rn-marginTop-1mnahxq rn-marginRight-61z16t rn-marginBottom-p1pxzi rn-marginLeft-11wrixw rn-paddingTop-wk8lta rn-paddingRight-9aemit rn-paddingBottom-1mdbw0j rn-paddingLeft-gy4na3 rn-textDecoration-bauka4 rn-whiteSpace-q42fyq rn-wordWrap-qvutc0"
            dir="auto"
          >
            hello
          </div>
        </Text>
      </tr>
    </thead>
  </GetTheme>
</TableHeader>
`;
